<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditPassword</class>
 <widget class="QDialog" name="DialogEditPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="plblNewPassword">
         <property name="text">
          <string>Старый пароль</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditOldPassword">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	border: 2px solid gray;
     border-radius: 4px;
     padding: 0 8px;
	background-color: rgb(225, 225, 225);
	color: rgb(48, 48, 48);
    selection-background-color: rgb(0, 170, 255);
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="plblOldPassword">
         <property name="text">
          <string>Новый пароль</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditNewPassword">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	border: 2px solid gray;
     border-radius: 4px;
     padding: 0 8px;
	background-color: rgb(225, 225, 225);
	color: rgb(48, 48, 48);
    selection-background-color: rgb(0, 170, 255);
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="plblConfirmPassword">
         <property name="text">
          <string>Повторите пароль</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditConfirmPassword">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	border: 2px solid gray;
     border-radius: 4px;
     padding: 0 8px;
	background-color: rgb(225, 225, 225);
	color: rgb(48, 48, 48);
    selection-background-color: rgb(0, 170, 255);
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkBoxShowPassword">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Показать пароль</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtnOk">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:enabled {
  	background-color: rgb(194, 55, 0);
  	color: white;
	border-radius: 5px;
    border-color: beige;
	min-width: 70px;
	min-height: 22px;
}

QPushButton {
  	background-color: rgb(98, 98, 98);
  	color: white;
	border-radius: 5px;
    border-color: beige;
	min-width: 60px;
	min-height: 20px;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(255, 125, 11), stop:1 rgb(255, 188, 17))
}

QPushButton:pressed {
     background-color: rgb(98, 98, 98);
     border-style: inset;
 }</string>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbrnCancel">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:enabled {
  	background-color: rgb(194, 55, 0);
  	color: white;
	border-radius: 5px;
    border-color: beige;
	min-width: 70px;
	min-height: 22px;
}

QPushButton {
  	background-color: rgb(98, 98, 98);
  	color: white;
	border-radius: 5px;
    border-color: beige;
	min-width: 60px;
	min-height: 20px;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(255, 125, 11), stop:1 rgb(255, 188, 17))
}

QPushButton:pressed {
     background-color: rgb(98, 98, 98);
     border-style: inset;
 }</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbrnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogEditPassword</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
